edx_solutions_organizations/views.py

```diff
@@ -120,6 +120,20 @@ def retrieve(self, request, *args, **kwargs):
         self.serializer_class = BasicOrganizationSerializer
         return super().retrieve(request, *args, **kwargs)
 
+    @detail_route(methods=['get', ])
+    def switch_docebo_redirect_flag(self, request, pk):
+        organization = Organization.objects.get(id=pk)
+        organization.is_redirected_to_docebo = not organization.is_redirected_to_docebo
+        organization.save()
+        return Response({"message": "Organization {} updated".format(pk)}, status=status.HTTP_200_OK)
+
+    @detail_route(methods=['get', ])
+    def switch_sso_enabled_flag(self, request, pk):
+        organization = Organization.objects.get(id=pk)
+        organization.is_sso_enabled = not organization.is_sso_enabled
+        organization.save()
+        return Response({"message": "Organization {} updated".format(pk)}, status=status.HTTP_200_OK)
```

Comment:
We don't need to add these methods. This DRF API supports patch method where we can pass the fields to update. 
See the API patch being called from Apros: https://github.com/mckinseyacademy/mcka_apros/blob/master/courses/tasks.py#L112